roachtest: hibernate failed

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10627975?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10627975?buildTab=artifacts#/hibernate) on release-23.1 @ [3532484aa08c3acf6eb5770f1469af09f4ba704d](https://github.com/cockroachdb/cockroach/commits/3532484aa08c3acf6eb5770f1469af09f4ba704d):


```
test artifacts and logs in: /artifacts/hibernate/run_1
(hibernate.go:167).func1: all attempts failed for building hibernate (without tests): output in run_112741.916618519_n1_cd-mntdata1hibernate: cd /mnt/data1/hibernate/hibernate-core/ && ./../gradlew test -Pdb=cockroachdb --tests org.hibernate.jdbc.util.BasicFormatterTest.* returned: COMMAND_PROBLEM: ssh verbose log retained in ssh_112741.921399245_n1_cd-mntdata1hibernate.log: exit status 1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99201 roachtest: hibernate failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
- #97842 roachtest: hibernate failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'hibernate'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find nu.studer:java-ordered-properties:1.0.2.
     Searched in the following locations:
       - https://jcenter.bintray.com/nu/studer/java-ordered-properties/1.0.2/java-ordered-properties-1.0.2.pom
       - https://repo.maven.apache.org/maven2/nu/studer/java-ordered-properties/1.0.2/java-ordered-properties-1.0.2.pom
       - https://plugins.gradle.org/m2/nu/studer/java-ordered-properties/1.0.2/java-ordered-properties-1.0.2.pom
     Required by:
         project : > nu.studer.credentials:nu.studer.credentials.gradle.plugin:2.1 > nu.studer:gradle-credentials-plugin:2.1

* Try:
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings
```

--- a/pkg/cmd/roachtest/tests/hibernate.go
+++ b/pkg/cmd/roachtest/tests/hibernate.go
@@ -23,7 +23,7 @@ import (
 )
 
 var hibernateReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var supportedHibernateTag = "5.6.9"
+var supportedHibernateTag = "5.6.15"
 
 type hibernateOptions struct {
 	testName string

